libraries/DataFlash/LogFile.cpp

```diff
@@ -2086,3 +2086,16 @@ void DataFlash_Class::Log_Write_Rally(const AP_Rally &rally)
         }
     }
 }
+
+// Write AOA and SSA
```

Comment:
this works, but I think we can use the more compact Log_Write() method on DataFlash for msgs like this